app/controllers/sales_controller.rb

```diff
@@ -38,7 +38,13 @@ def new
     @sale.customer_id = params[:customer_id] if params[:customer_id]
   end
 
+  # returnpath from adding comments
+  def edit
+    redirect_to sale_path(@sale)
+  end
+
```

Comment:
It must be possible to direct to the `#show` method directly without going to `#edit` first.